app/src/main/java/com/example/wnews/views/root/RootActivity.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.wnews.views.root
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.preference.PreferenceManager
+import com.example.wnews.R
+import com.example.wnews.views.auth.AuthPresenter
+import com.example.wnews.views.auth.login.LoginActivity
+import com.example.wnews.views.main.MainActivity
+
+class RootActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_root)
```

Comment:
I think that the Root Activity may be left without a layout, as it shouldn't display anything visually.